examples/dispatch-mouse-event/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+    <div>
+        <button id="target">Target button</button>
+        <label>Seconds pressed: </label>
+        <span id="timer">0</span>
+    </div>
+
+</body>
+<script defer>
+    const btn = document.getElementById('target');
```

Comment:
Align assignments.

Reply:
It's time to add the code linting to the `testcafe-examples` repository.
Copy the linting from here - https://github.com/DevExpress/testcafe-react-selectors/blob/master/Gulpfile.js#L25

Reply:
in a [new PR](https://github.com/DevExpress/testcafe-examples/pull/53)